### PR TITLE
server,config: add support to ignore cloning certain labels

### PR DESCRIPTION
**Pull request by AlexNPavel**

### CODE REVIEW

File: cmd/jira-lifecycle-plugin/server.go [hunk 1]
```diff
@@ -1826,8 +1826,11 @@ refIssueLoop:
 		// GetIssue is invalid for setting the field ourselves
 		sprintField := bug.Fields.Unknowns[helpers.SprintField]
 		delete(bug.Fields.Unknowns, helpers.SprintField)
-		releaseNoteType := bug.Fields.Unknowns[helpers.ReleaseNoteTypeField]
-		releaseNoteText := bug.Fields.Unknowns[helpers.ReleaseNoteTextField]
```
Comment (bradmwilliams) [inline cmd/jira-lifecycle-plugin/server.go:1830]: Did you mean to delete the lines associated with ReleaseNotes?

File: cmd/jira-lifecycle-plugin/server.go [hunk 2]
```diff
@@ -1869,12 +1872,6 @@ refIssueLoop:
 				},
 			},
 		}
-		if releaseNoteText != nil {
-			update.Fields.Unknowns[helpers.ReleaseNoteTextField] = releaseNoteText
-		}
-		if releaseNoteType != nil {
-			update.Fields.Unknowns[helpers.ReleaseNoteTypeField] = releaseNoteType
-		}
```
Comment (bradmwilliams) [inline cmd/jira-lifecycle-plugin/server.go:1877]: Same question as above

**Review by bradmwilliams — CHANGES_REQUESTED** [review]
Looks good, but I want to make sure a couple changes were intentional!

File: cmd/jira-lifecycle-plugin/server.go (hunk 1 above)
Comment (AlexNPavel) [reply cmd/jira-lifecycle-plugin/server.go]: Not sure how that happened (I think I may have checked out an older version of server.go before the release note changes?). Should be good now

**Review by bradmwilliams — APPROVED** [review]

**Pull request merged**